src/frontend/src/components/public/PIAFormTabs/PIAStoredPersonalInformation/messages.ts

```diff
@@ -0,0 +1,113 @@
+export default {
+  Heading: {
+    H2Text: {
+      en: `Storing Personal Information`,
+    },
+    PText: {
+      en: `If you're storing personal information outside of Canada, identify the sensitivity of the personal information and where and how it will be stored.`,
+    },
+  },
+  SectionOne: {
```

Comment:
I've used this naming convention so that if the content changes (i.e.: the heading text), the naming here will not have to change. I find that naming these too specifically means that I will have to make changes in multiple places.

Reply:
I see. It feels like we will have to make some weighted choice in that case.
I think changing the order could be more frequent than the question itself. Like in a recent change from Product inside the Personal Information Banks tab, we are asked to add a new section ahead of the existing one. Had we got sections numbered, like sectionOne there, it would have been super confusing after implemention,